app/controllers/api/v4/converts_controller.rb

```diff
@@ -33,7 +33,8 @@ def create
   private
 
   def check_parameters
-    params.require(:file, :format)
+    params.require(:file)
+    params.require(:format)
```

Comment:
Weird, does an array work? [The docs]http://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-require) say that an array works but their example is of two arguments, so maybe the example is wrong.